app/user_test.go

```
@@ -261,6 +263,23 @@ func TestUpdateActiveBotsSideEffect(t *testing.T) {
 	th.App.UpdateActive(th.BasicUser, true)
 }
 
+func TestUserDeleteUpdateDirectChannelDeleted(t *testing.T) {
+	th := Setup(t).InitBasic()
+	defer th.TearDown()
+
+	th.CreateDmChannel(th.BasicUser2)
+	basicUser3 := th.CreateUser()
+	th.CreateDmChannel(basicUser3)
+
+	th.App.UpdateActive(th.BasicUser, false)
+	require.NotZero(t, th.BasicUser.DeleteAt)
+
+	directChannels, _ := th.App.Srv().Store.Channel().GetDirectChannelsForUser(th.BasicUser.Id)
```

We should not ignore the error.